js/templates/serverConnectModal.html

```diff
@@ -0,0 +1,168 @@
+<div class="modal-hero modal-shadow-inner1 bg-dark-blue custCol-primary-darken">
+  <div class="modal-hero-icon"></div>
+  <% if (ob.status == 'trying') { %>
+    <p class="h6 homeModal-heroText fancy-heading"><%= polyglot.t('serverConnectModal.statusTryingToConnect') %>...</p>
+  <% } else if (ob.status == 'connected') { %>
+    <p class="h6 homeModal-heroText fancy-heading"><%= polyglot.t('serverConnectModal.statusConnected') %>.</p>  
+  <% } else if (ob.status == 'failed') { %>  
+    <p class="h6 homeModal-heroText fancy-heading"><%= polyglot.t('serverConnectModal.statusFailedConnection') %>.</p>
+  <% } %>
+  </p>
+</div>
+<div class="accordion-window" style="left: 0px;">
+  <div class="accordion-child accordion-active custCol-primary overflowHidden" style="width: 525px; height: 374px;">
+    <div class="bar barTxt h6 custCol-secondary custCol-text">
+      <%= polyglot.t('serverConnectModal.serverConfiguration') %>
+      <span class="textOpacity75 fontSize14 floatRight">
+        <% if (ob.status == 'trying') { %>
+          <!-- variablize the max attempts -->
+          <i class="ion-android-sync spinner"></i>
+          <% if (ob.isLocal) { %>
+          <%= polyglot.t('serverConnectModal.attempt') %> <span class="attempt"></span> <%= polyglot.t('of') %> 3
+          <% } %>
+        <% } else if (ob.status === 'failed') { %>
+        <a class="js-retry"><%= polyglot.t('serverConnectModal.retry') %></a>
+        <% } %>
+      </span>
+    </div>
+    <div class="flexContainer scrollOverflowYHideX customThemeScrollbar" style="height: 272px">
+      <div class="flexRow borderBottom custCol-border">
+        <% if (!ob.isLocal) { %>
+        <p class="margin0 padding015 marginBottom20">We are having difficulties establishing a connection with your server. The two most likely reasons are:</p>
```

Comment:
Haven't polyglotted the messaging for remote servers, because it didn't come from the design and I imagine it's going to change.